src/types/publicationFieldNames.ts

```diff
@@ -44,7 +44,7 @@ export const instanceTypeBaseFieldName = 'entityDescription.reference.publicatio
 export enum ReferenceFieldNames {
   ARTICLE_NUMBER = 'entityDescription.reference.publicationInstance.articleNumber',
   DOI = 'entityDescription.reference.doi',
-  IS_TEXTBOOK = 'entityDescription.reference.publicationInstance.isTextbook',
+  IS_TEXTBOOK = 'entityDescription.reference.publicationInstance.textbookContent',
```

Comment:
```suggestion
  TEXTBOOK_CONTENT= 'entityDescription.reference.publicationInstance.textbookContent',
```

Reply:
Jeg synes egentlig `is textbook` er mer beskrivende enn `textbookContent`, så foretrekker at vi bruker det. Synes du det er bedre med textbookContent?

Reply:
men samtidig er det kun brukt en plass da, så kan godt rename

Reply:
syns det er enklere om vi bruker samme navn som backend, hvis ikke blir det fort forvirring om det er samme felt eller ikke